alpha/apps/kaltura/lib/reports/kKavaReportsMgr.class.php

```diff
@@ -7815,4 +7827,65 @@ protected static function formatTimestampToDay(&$result, $input_filter)
 			$row[$timePosition] = $timestamp;
 		}
 	}
+
+	protected static function sendAnalyticsBeacon($host, $port, $partner_id, $id, $params)
```

Comment:
why not using the existing sendAnalyticsBeacon on alpha/apps/kaltura/lib/requestUtils.class.php:337
its seems like this function is explicitly for custom reports beacons